File: comms/bulk-messaging/sender-pools/add-sender-to-pool/add-sender-to-pool.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "<username>",
        authToken: "<password>",
    });
    await client.senderPools.addSender("comms_senderpool_01h9krwprkeee8fzqspvwy6nq8", [
        {
            senderId: "comms_sender_01h9krwprkeee8fzqspvwy6nq8",
        },
    ]);
}
main();

File: comms/bulk-messaging/sender-pools/add-sender-to-pool/add-sender-to-pool.rb
require "twilio"

client = Twilio::Client.new(
  account_id: "<username>",
  auth_token: "<password>"
)

client.sender_pools.add_sender(
  sender_pool_id: "comms_senderpool_01h9krwprkeee8fzqspvwy6nq8",
  request: [{
    sender_id: "comms_sender_01h9krwprkeee8fzqspvwy6nq8"
  }]
)
